observability/monitoring/application-monitoring.adoc

```diff
@@ -14,7 +14,3 @@ include::snippets/technology-preview.adoc[leveloffset=+0]
 include::modules/monitoring-configuring-cluster-for-application-monitoring.adoc[leveloffset=+1]
 include::modules/monitoring-configuring-monitoring-for-an-application.adoc[leveloffset=+1]
 include::modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc[leveloffset=+1]
-
-== Next steps
-
-* To automatically adjust the number of pods in which the application runs, xref:../../observability/monitoring/application-monitoring.adoc#application-monitoring[configure Horizontal Pod Autoscaling for the application.]
```

Comment:
did not find observability/monitoring/application-monitoring.adoc for 4.12-4.16, can you help to list the doc URL?

Reply:
This is an unused chapter AFAIK, I made these changes in there just in case :smiley: (I cannot delete this from the repo, but it might get deleted eventually)